docs/datadog_checks_base.rst

```diff
@@ -10,7 +10,6 @@ Subpackages
     :maxdepth: 3
 
     datadog_checks.checks
-    datadog_checks.stubs
     datadog_checks.utils
```

Comment:
Let's change everything to `datadog_checks.base.`

Reply:
done 👍